infor-design/enterprise-wc
# 576 - Fix dirty tracker rtl and other stuff

**Explain the details for making this change. What existing problem does the pull request solve?**
- Fixed dirty tracker mixin for RTL language was not working properly.
- Fixed Spinbox for disabled and all RTL styles were missing
- Fixed lookup selection was not working

**Related github/jira issue (required)**:
Closes #576

**Steps necessary to review your pull request (required)**:
**Steps necessary to review your pull request (required)**:
- Pull this branch and build/run the demo app
- Navigate to:
- http://localhost:4300/ids-checkbox
- http://localhost:4300/ids-color-picker
- http://localhost:4300/ids-date-picker
- http://localhost:4300/ids-dropdown
- http://localhost:4300/ids-editor/sandbox.html
- http://localhost:4300/ids-input
- http://localhost:4300/ids-lookup
- http://localhost:4300/ids-radio
- http://localhost:4300/ids-spinbox
- http://localhost:4300/ids-textarea
- http://localhost:4300/ids-time-picker
- http://localhost:4300/ids-upload
- Test for `Dirty Tracking` example
- It should add/remove yellow triangle icon
- Change to some RTL language, for example `ar`
- See the `Dirty Tracking` icon should working properly

Spinbox RTL styles
- Navigate to: http://localhost:4300/ids-spinbox
- Change to some RTL language, for example `ar`
- See all the styles, it should not misaligned

Lookup selection
- Navigate to: http://localhost:4300/ids-lookup
- Open lookup modal
- Should be able to select/unselect

**Included in this Pull Request**:
